src/main/java/org/spongepowered/common/data/key/KeyRegistry.java

```diff
@@ -1,74 +1,76 @@
-/*
- * This file is part of Sponge, licensed under the MIT License (MIT).
```

Comment:
Yeah.. this shouldn't be happening. please fix your formatting changes. i.e. don't change our formatting.